src/python/pants/option/options.py

```diff
@@ -226,6 +226,14 @@ def unknown_goals(self) -> list[str]:
     def known_scope_to_info(self) -> dict[str, ScopeInfo]:
         return self._known_scope_to_info
 
+    @property
+    def known_scope_to_args(self) -> dict[str, frozenset[str]]:
+        return {scope: parser.known_args for scope, parser in self._parser_by_scope.items()}
```

Comment:
Is this used?

Reply:
No, it's just to have the symmetric pair. I'll remove it.